src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -121,26 +121,25 @@ static class LeaderboardView<TEntry> where TEntry : WebApi.IEntry
                 ColorConsole.WriteLine();
                 foreach (var line in lines)
                 {
-                    ColorConsole.WriteLine(new ColorToken[] { padding }.Concat(line).ToArray());
+                    ColorConsole.WriteLine(new ColorToken[]
+                    {
+                        padding
+                    }.Concat(line).ToArray());
                 }
 
                 ColorConsole.WriteLine(
                     padding,
-                    $"Total {itemsName}:".Gray(),
-                    " ",
-                    $"{currentLeaderboard?.Count ?? 0:N0}"
-                        .Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]),
-                    $" since ".Gray(),
-                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")}".Gray());
+                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")} to {currentLeaderboard.LastActivityDate?.ToLocalTime().ToString() ?? ("started")}".Gray());

```

Comment:
Ok I get it now. So I recomend removing this line in case of null, see below:
![image](https://cloud.githubusercontent.com/assets/604826/11719619/dba092b4-9f5b-11e5-9816-a5158bf506f7.png)